make.js

```diff
@@ -317,27 +305,27 @@ target.build = function() {
 // node make.js test
 // node make.js test --task ShellScript --suite L0
 //
-target.test = function() {
+CLI.test = function(/** @type {{ suite: string; node: string; task: string; … }} */ argv) {
```

Comment:
Please ditch the `…`, that was from my example JSDoc notation to indicate more fields can be added :)